### DISCUSSION TITLE
GitHub Actions Automations

### DISCUSSION THREAD

**Discussion by fishcharlie** [blocklistproject/Lists#284] [Ideas]
Recently the Everything list was deprecated.

Would this project be interested in using GitHub Actions to help automate some tasks? For example we could build a script to automatically generate the Everything list on every commit. We could also extend this to the no leading IP lists. It could also be expanded to automatically sort the lists.

I see a lot of potential for automation in this project, is this something that people would be interested in considering?

If there is interest in it, I might be able to find some free time to get something up and running in regards to this.

**Comment by blocklistproject** [comment]
That would be awesome! I am still learning everything that GH can do and any help would be much appreciated!

**Comment by fishcharlie** [comment]
@blocklistproject Awesome. I'll see if I can find some time sometime soon to work on this.

**Comment by blocklistproject** [comment]
@fishcharlie I am going to send you a DM as well

**Comment by fishcharlie** [comment]
@blocklistproject All my contact information can be found at https://charlie.fish/contact

**Comment by blocklistproject** [comment]
@fishcharlie Just sent you a message on FB messenger
